beacon_node/beacon_processor/src/lib.rs

```diff
@@ -1001,31 +629,20 @@ impl<T: BeaconChainTypes> Stream for InboundEvents<T> {
     }
 }
 
-/// Defines if and where we will store the SSZ files of invalid blocks.
-#[derive(Clone)]
-pub enum InvalidBlockStorage {
```

Comment:
`InvalidBlockStorage` was moved to the `beacon_node/network` crate. It's not relevant to the `BeaconProcessor`.